### PR TITLE
Update Nethermind Client info

### PR DESCRIPTION
**Pull request by smartprogrammer93**
## Description
Simply updating information about the Nethermind Client.

### CODE REVIEW

File: src/content/developers/docs/nodes-and-clients/index.md [hunk 1]
```diff
@@ -208,9 +208,10 @@ Light client mode downloads all block headers, block data, and verifies some ran
 
 ##### Snap sync
 
-Implemented by Geth. Using dynamic snapshots served by peers retrieves all the account and storage data without downloading intermediate trie nodes and then reconstructs the Merkle trie locally.
+Implemented by Geth and Nethermind. Using dynamic snapshots served by peers retrieves all the account and storage data without downloading intermediate trie nodes and then reconstructs the Merkle trie locally.
 
-- Fastest sync strategy developed by Geth, currently its default
+- Fastest sync strategy developed by Geth, currently its default.
+- Nethermind only implemented the consuming part till now. They working on enabling serving other nodes soon.
```
Comment (corwintines) [inline src/content/developers/docs/nodes-and-clients/index.md:214]:
```suggestion
- Nethermind only implemented the consuming part until now. They are working on enabling serving other nodes soon.
```

**Review by corwintines — COMMENTED** [review]
Thanks, @smartprogrammer93! Just made a suggestion for a copy edit.

File: src/content/developers/docs/nodes-and-clients/index.md [hunk 2]
```diff
@@ -123,13 +123,13 @@ The Ethereum community maintains multiple open-source execution clients (previou
...
-| [Erigon](https://github.com/ledgerwatch/erigon) | Go       | Linux, Windows, macOS | Mainnet, Görli, Rinkeby, Ropsten           | Full                | Archive, Pruned |
-| [Akula](https://akula.app)                      | Rust     | Linux                 | Mainnet, Görli, Rinkeby, Ropsten, and more | Full                | Archive, Pruned |
+| Client                                          | Language | Operating systems     | Networks                                   | Sync strategies                   | State pruning   |
+| ----------------------------------------------- | -------- | --------------------- | ------------------------------------------ | ----------------------------------| --------------- |
+| [Geth](https://geth.ethereum.org/)              | Go       | Linux, Windows, macOS | Mainnet, Görli, Rinkeby, Ropsten           | Snap, Full                        | Archive, Pruned |
+| [Nethermind](http://nethermind.io/)             | C#, .NET | Linux, Windows, macOS | Mainnet, Görli, Ropsten, Rinkeby, and more | Snap(without serving), Fast, Full | Archive, Pruned |
```
Comment (minimalsm) [inline src/content/developers/docs/nodes-and-clients/index.md:129]:
```suggestion
| [Nethermind](http://nethermind.io/)             | C#, .NET | Linux, Windows, macOS | Mainnet, Görli, Ropsten, Rinkeby, and more | Snap (without serving), Fast, Full | Archive, Pruned |
```

**Review by jmcook1186 — APPROVED** [review]
Ok - thanks for updating this!

**Comment by corwintines** [comment]
@all-contributors can you add @smartprogrammer93 for docs

**Comment by corwintines** [comment]
Hey @smartprogrammer93 , thanks for the PR :)

Be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And we've just released our 2022 POAPs so remember to [claim yours](https://ethereum.org/en/contributing/#poap) also!

**Pull request merged**